visualization/include/pcl/visualization/common/common.h

```diff
@@ -153,6 +156,13 @@ namespace pcl
           * size are initialized with (0, 0) and (1, 1) respectively. */
         Camera ();
 
+        /** Construct a camera by copying parameters from a VTK camera.
+          * Window position and size are initialized with (0, 0) and (1, 1) respectively.*/
+        Camera (vtkCamera& camera);
```

Comment:
Shouldn't this be a const reference?

Reply:
Unfortunately their getters are not `const`-qualified... At least not in VTK 6.

Reply:
Hmm. Should we add some remarks in the docstring to transition to a const reference in case VTK const qualifies the getters?

Reply:
Here is one of the macros they use to create getters:

https://github.com/Kitware/VTK/blob/7bf6ed64170da1a7adf8607b80675bf16b281403/Common/Core/vtkSetGet.h#L314-L319

As you can see, to date they don't have `const`, so I don't think in any foreseeable future we will have a baseline VTK version with `const`.

Reply:
Thank you for digging that up.